ddclient

```diff
@@ -1708,10 +1708,7 @@ sub opt {
     my $v = shift;
     my $h = shift;
     return $config{$h}{$v} if defined($h) && defined($config{$h}{$v});
-    return $opt{$v}        if defined $opt{$v};
-    return $globals{$v}    if defined $globals{$v};
-    return default($v)     if defined default($v);
-    return undef;
+    return $opt{$v} // $globals{$v} // default($v);
```

Comment:
Does this still return undef?

Reply:
Yes. Note that:

```perl
    return default($v)     if defined default($v);
    return undef;
```

is equivalent to:

```perl
    return default($v);
```